pkg/reloader/reloader.go

```diff
@@ -98,6 +99,13 @@ func (r *Reloader) Run(ctx context.Context) error {
 		return err
 	}
 
+	// If we have an auth file configuration, watch it also.
+	if r.AuthFile != "" {
+		if err := configWatcher.Add(r.AuthFile); err != nil {
+			return err
+		}
+	}
```

Comment:
actually yes this is needed because it takes a digest of the file contents, and only sends signal when it is different...